src/react-apps/applications/service-development/src/App.tsx

```diff
@@ -2,232 +2,115 @@
 // https://github.com/facebook/create-react-app/issues/4801#issuecomment-409553780
 // Disabled for React Router rendering
 
+/* tslint:disable:jsx-boolean-value */
+// Extensive used in Material-UI's Grid
+
 import Grid from '@material-ui/core/Grid';
+import Hidden from '@material-ui/core/Hidden';
+import { createMuiTheme, createStyles, MuiThemeProvider, withStyles, WithStyles } from '@material-ui/core/styles';
 import * as React from 'react';
-import { connect } from 'react-redux';
+import { HashRouter as Router, Redirect, Route } from 'react-router-dom';
 import LeftDrawerMenu from '../../shared/src/navigation/drawer/LeftDrawerMenu';
-import AppBarComponent from '../../shared/src/navigation/main-header/app-bar';
-import SubApp from '../../ux-editor/src/SubApp';
+import AppBarComponent from '../../shared/src/navigation/main-header/appBar';
+import altinnTheme from '../../shared/src/theme/altinnStudioTheme';
 import NavigationActionDispatcher from './actions/navigationActions/navigationActionDispatcher';
 import './App.css';
+import { redirects } from './config/redirects';
+import { routes } from './config/routes';
 
-import { HashRouter as Router, Redirect, Route } from 'react-router-dom';
+export interface IAppProps { }
 
-const DummySubApp = (name: any) => {
-  return (
-    <div>Dummy app for {name.name}</div>
-  );
-};
+const theme = createMuiTheme(altinnTheme);
 
-export interface IAppProps {
-  drawerOpen: boolean;
-}
+const styles = () => createStyles({
+  subApp: {
+    [theme.breakpoints.up('md')]: {
+      paddingLeft: 100,
+    },
+  },
+});
 
-class AppClass extends React.Component<IAppProps, any> {
+class App extends React.Component<IAppProps & WithStyles<typeof Object>, any> {
```

Comment:
Since this App-class doesn't receive any props, it should be 
`class App extends React.Component<null & WithStyles<object>`.

If `null` isn't working, use `any`.  And remove the interface.

Reply:
Fixed, Removed the IAPPProps